README.md

```diff
@@ -282,3 +282,21 @@ rm ~/.zcompdump*
 ```
 
 Then start a new shell or run `source ~/.zshrc` once.
+
+## Verify kwctl binaries
+
+kwctl binaries are signed using [Sigstore](https://docs.sigstore.dev/cosign/working_with_blobs/#signing-blobs-as-files). 
+Each release contains two files: kwctl.sig and kwctl.pem that can be used for verification.
```

Comment:
How do I download these files and make sure they are not tampered?

Can you provide some instructions about that?

Reply:
these file are inside the .zip file in the release, see example https://github.com/raulcabello/kwctl/releases/tag/v0.0.0.888
I explained it here https://github.com/kubewarden/kwctl/pull/230/commits/c57eacfae3dedff9f4b32f109930f086ea9b7fba
we should probably do a new kwctl release, so .sig and .pem files are available